### PR TITLE
Hlu/add common transformers utils

### PR DESCRIPTION
**Pull request by hlums**
### Checklist:

- [ ] My code follows the code style of this project, as detailed in our [contribution guidelines](../CONTRIBUTING.md).
- [ ] I have added tests.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: utils_nlp/models/transformers/common.py [hunk 1]
```diff
@@ -1,4 +1,16 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
 
+from pytorch_transformers.modeling_bert import BERT_PRETRAINED_MODEL_ARCHIVE_MAP
+from pytorch_transformers.modeling_xlnet import XLNET_PRETRAINED_MODEL_ARCHIVE_MAP
+from pytorch_transformers.modeling_roberta import ROBERTA_PRETRAINED_MODEL_ARCHIVE_MAP
+
+BERT_PRETRAINED_MODEL_ALL = list(BERT_PRETRAINED_MODEL_ARCHIVE_MAP.keys())
```
Comment (awaemmanuel) [inline utils_nlp/models/transformers/common.py]: Looks good just remove list(..). Dictionary `keys()` already return a list

Comment (hlums) [reply utils_nlp/models/transformers/common.py]: Dictionary keys() returns dict_keys type. It's iterable, but a regular list is probably better.

Comment (saidbleik) [reply utils_nlp/models/transformers/common.py]: you can remove the .keys() though :)

Comment (hlums) [reply utils_nlp/models/transformers/common.py]: Wow, why is that possible? I think I'm going to close this PR for now and do something like you did in the seq classifier PR.

Comment (saidbleik) [reply utils_nlp/models/transformers/common.py]: a list can take an iterable object, and converts it to a list by accessing the __iter__ function, which in the case of dictionaries, returns the keys

**Pull request closed without merging**